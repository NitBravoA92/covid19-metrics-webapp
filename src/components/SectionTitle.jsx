import PropTypes from 'prop-types';

const SectionTitle = ({ title }) => (
  <h2 className="section-title">{ title }</h2>
);

SectionTitle.propTypes = {
  title: PropTypes.string.isRequired,
};

export default SectionTitle;
