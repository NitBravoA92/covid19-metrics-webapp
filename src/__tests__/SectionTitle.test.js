import '@testing-library/jest-dom/extend-expect';
import { render, screen } from '@testing-library/react';
import SectionTitle from '../components/SectionTitle';

describe('The SectionTitle component', () => {
  test('should render correctly into the DOM', () => {
    const { sectionTitleComponent } = render(
      <SectionTitle title="STATS BY CONTINENTS" />,
    );
    expect(sectionTitleComponent).toMatchSnapshot();
  });

  test('should render with the text: STATS BY COUNTRIES', () => {
    const longTitle = 'STATS BY COUNTRIES';
    render(<SectionTitle title={longTitle} />);
    const sectionTitle = screen.getByText(longTitle);
    expect(sectionTitle).toBeInTheDocument();
  });
});
