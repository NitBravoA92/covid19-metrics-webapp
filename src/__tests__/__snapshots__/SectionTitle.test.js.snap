// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The SectionTitle component should render correctly into the DOM 1`] = `undefined`;
